datadisplay.cpp

```diff
@@ -59,6 +59,9 @@ DataDisplay::DataDisplay(QWidget *parent)
 
     connect(m_searchPanel, &SearchPanel::findNext, this, &DataDisplay::find);
     connect(m_searchPanel, &SearchPanel::textEntered, m_highlighter, &DataHighlighter::setSearchString);
+
+    connect(&m_timer, SIGNAL(timeout()), this, SLOT(BlockReady()));
+    m_timer.start(100);
```

Comment:
Thank you for the comments, I will add your corrections to my branch. As I'm new to GitHub, please answer, should I do corrections as new commit, or may I make a fixup using git rebase -i and push?